### PR TITLE
ASoC: SOF: ipc4-topology: Use macro to set the EXT_PARAM_SIZE in widg…

### PR DESCRIPTION
**Pull request by ujfalusi**
…et setup

Use the SOF_IPC4_MOD_EXT_PARAM_SIZE() macro to set the param size in the extension part of the IPC message for clarity.

### CODE REVIEW

**Review by bardliao — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]

File: sound/soc/sof/ipc4-topology.c [hunk 1]
```diff
@@ -2828,7 +2828,7 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 		msg->primary |= SOF_IPC4_MOD_INSTANCE(swidget->instance_id);
 
 		msg->extension &= ~SOF_IPC4_MOD_EXT_PARAM_SIZE_MASK;
-		msg->extension |= ipc_size >> 2;
+		msg->extension |= SOF_IPC4_MOD_EXT_PARAM_SIZE(ipc_size >> 2);
```
Comment (ranj063) [inline sound/soc/sof/ipc4-topology.c:2831]: this has no effect and neither does this fix anything right? Can we add that in the commit message?

Comment (ujfalusi) [reply sound/soc/sof/ipc4-topology.c:2831]: OK, it is just clarifying the sloppy implementation, no functional change.

Comment (lgirdwood) [reply sound/soc/sof/ipc4-topology.c:2831]: @ujfalusi ping.

**Comment by ujfalusi** [comment]
Changes since v1:
- added explicit clarification of no functional change to commit message

**Review by lgirdwood — APPROVED** [review]

**Review by ranj063 — APPROVED** [review]

**Pull request merged**